Scientific_workflows/DEAWaterbodies/DEAWaterbodiesToolkit/ENSOInfluence.ipynb

```diff
@@ -48,11 +48,20 @@
    "outputs": [],
```

Comment:
<p>Consider replacing "early winter" with "late autumn/early winter (MAM/JJA)" - adds the seasonal acronym, for context for those whom winter is hard to place :D </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/724/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>